modules/core/src/main/java/org/apache/ignite/internal/processors/cluster/GridClusterStateProcessor.java

```diff
@@ -924,49 +945,8 @@ protected IgniteCheckedException concurrentStateChangeError(ClusterState state,
 
     /** {@inheritDoc} */
     @Override public IgniteInternalFuture<?> changeGlobalState(
-        final boolean activate,
```

Comment:
Why this deletion?

Reply:
> Why this deletion?

To simplify and reduce internal API and code base. There were already 2 changeGlobalState(). First, I introduced new one with ‘force’ not to change previous API. There 3 (2 of them deprecated) became a mesh. After all, as it is internal API, I suggest to keep one change-state method. Isn’t it better?